plugins/reindex/src/main/java/org/elasticsearch/plugin/reindex/AbstractBulkByScrollRequest.java

```diff
@@ -72,6 +74,17 @@
      */
     private WriteConsistencyLevel consistency = WriteConsistencyLevel.DEFAULT;
 
+    /**
+     * Initial delay after a rejection before retrying request. Defaults to 50ms. With the default maxRetries to total time to retry
+     * rejects is about 25 seconds per rejection.
+     */
+    private TimeValue retryBackoffInitialTime = timeValueMillis(50);
+
+    /**
+     * Total number of retries attempted for rejections. Defaults to 10.
+     */
+    private int maxRetries = 10;
```

Comment:
> I'm not sure that negative numbers actually break vint.

Yeah, they don't break it, they are just more inefficient, using `writeVInt` to me is kind of an indication that you never expect a value to be negative (from the code-side), so it's nice to encapsulate what you expect for the intended values. Could always change this to `writeInt` instead